video.c

```diff
@@ -208,8 +209,10 @@ video_read_data_byte(struct video_struct* p_video,
     /* In the corner-case combination of MODE7 style addressing, plus a 2MHz
      * CRTC clock, a quirk of the memory refresh system is revealed. The
      * memory fetch address bit MA6 is xor'ed with the 1MHz clock.
+     * Testing LinearTTX.ssd on a BBC Master reveals that this xor of bit 6 should
```

Comment:
I think the word "should" spills past 80 columns? Yeah, sorry, it's just that that's the style. For better or worse.

Reply:
Reformatted to fix under the 80-char limit.